CONTRIBUTING.md

```diff
@@ -128,7 +128,7 @@ PYTHON_VERSION=3.13 ./scripts/unit-tests.sh
 
 ## Running integration tests
 
-You can run integration tests following the instructions [here](tests/integration/README.md).
+You can run integration tests following the instructions [here](https://github.com/meta-llama/llama-stack/tree/main/tests/integration#llama-stack-integration-tests).
```

Comment:
Hmm, the problem here is that this is challenging if you have the repo locally and want to navigate to the integration tests. 

This basically values the website over the docs.

Reply:
Probably there's some post compilation step we need to think through.

Reply:
let's drop this for now and we can investigate how we can better handle it? do you have bandwidth to look into it?

Reply:
sure